### DISCUSSION TITLE
Go get inside Docker hangs

### DISCUSSION THREAD

**Discussion by doodlesbykumbi** [community/community#24991] [Codespaces]
<p>When <code>go get</code> is run inside a Docker container it hangs until timeout. This doesn’t happen on the workspace host.</p>
<p>Steps to reproduce:</p>
<pre><code class="lang-auto">$ docker run --rm -it golang:1.13-stretch go get github.com/aws/aws-sdk-go
go: github.com/aws/aws-sdk-go@v1.37.5: Get https://proxy.golang.org/github.com/aws/aws-sdk-go/@v/v1.37.5.mod: dial tcp: lookup proxy.golang.org on 8.8.4.4:53: read udp 172.17.0.2:56052-&gt;8.8.4.4:53: i/o timeout
</code></pre>

**Comment by jkeech** [comment]
<p>Thanks for reporting! We’re rolling out a fix now.</p>

**Comment by jkeech** [comment, marked as answer]
<p>This should be fully rolled out. You might need to suspend and resume your codespace to pick up the network config fix. Let us know if you continue to see issues!</p>
